SPECS/shadow-utils/pam.d/system-password

```diff
@@ -3,6 +3,12 @@
 # use sha512 hash for encryption, use shadow, and try to use any previously
 # defined authentication token (chosen password) set by any prior module
 password  requisite   pam_pwquality.so
-password  required    pam_unix.so       sha512 shadow try_first_pass
+
+# Security patch for msid: 157.5
+# Ensure password reuse is limited
+password  required    pam_unix.so       sha512 shadow try_first_pass remember=5
+
+# Password complexity must be enforced
+password requisite pam_cracklib.so ucredit=-1 lcredit=-2 dcredit=-1 ocredit=-1 retry=3 minlen=10
```

Comment:
This may be a bit disruptive to some users, maybe we want this in the post image steps as well.